src/main/kotlin/apply/ui/admin/administrators/AdministratorFormDialog.kt

```diff
@@ -0,0 +1,65 @@
+package apply.ui.admin.administrators
+
+import apply.application.AdministratorService
+import com.vaadin.flow.component.Component
+import com.vaadin.flow.component.button.Button
+import com.vaadin.flow.component.dialog.Dialog
+import com.vaadin.flow.component.html.H2
+import com.vaadin.flow.component.orderedlayout.FlexComponent
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout
+import com.vaadin.flow.component.orderedlayout.VerticalLayout
+import support.views.createContrastButton
+import support.views.createNotification
+import support.views.createPrimaryButton
+
+class AdministratorFormDialog(
+    private val administratorService: AdministratorService,
+    displayName: String,
+    reloadComponents: () -> Unit
+) : Dialog() {
+
+    private val title = H2("관리자 $displayName")
+    private val administratorForm = AdministratorForm()
```

Comment:
c: 멤버 변수의 타입을 지정하면 어떨까요?

```suggestion
    private val title: H2 = H2("관리자 $displayName")
    private val administratorForm: AdministratorForm = AdministratorForm()
```